css/src/components/dropdown.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .Dropdown-inputWrapper {
-  border-bottom: var(--border);
+  border-bottom: var(--spacing-xs) solid var(--stone-dark);
```

Comment:
Please use the CSS token available on this link https://mds.innovaccer.com/?path=/story/others-design-tokens-colors--colors